### ISSUE TITLE
No padding idx arg in gluon.nn.Embedding

### ISSUE THREAD

**Issue by lucidfrontier45** [apache/incubator-mxnet#9257]
How can I do equivalent for PyTorch's `padding_idx` or Chainer's `ignore_label` in gluon's Embedding? 
http://pytorch.org/docs/master/nn.html#torch.nn.Embedding
https://docs.chainer.org/en/stable/reference/generated/chainer.links.EmbedID.html#chainer.links.EmbedID

I don't see any similar argument in gluon doc.
https://mxnet.incubator.apache.org/api/python/gluon.html#mxnet.gluon.nn.Embedding

Thank you.

**Comment by indhub** [comment]
This is not an issue. Please use the [user forum](https://discuss.mxnet.io/) for quicker response for how-to questions.

**Comment by altosaar** [comment]
This would be a nice feature to have.
